docs/index.mdx

```diff
@@ -37,8 +37,8 @@ sustainable.
 - **Automate everything:** Testing is boring, updating is boring, deploying is
   boring, coming up with release numbers is _really_ boring. Bored humans make
   mistakes. Machines don't make mistakes and they don't complain (yet). So we
-  strife to free our brains from everything we can let machines do and put them
-  onto more interesting tasks.
+  strive to free our brains from everything we can let machines do and rather put 
+  our brains to work on more interesting tasks.
```

Comment:
can we avoid that double-"brains"? or is it a strategy to show how much of them we have? 😄

Reply:
Haha sure! I'll rework it

Reply:
@pmelab - And so we have minds, and braaaaains! ;)